src/main/java/com/amazon/opendistroforelasticsearch/security/securityconf/ConfigModelV7.java

```diff
@@ -996,9 +997,19 @@ public TenantHolder(SecurityDynamicConfiguration<RoleV7> roles, SecurityDynamicC
                         if (tenants != null) {
                             
                             for (RoleV7.Tenant tenant : tenants) {
-                                
-                                for(String matchingTenant: WildcardMatcher.from(tenant.getTenant_patterns()).getMatchAny(definedTenants.getCEntries().keySet(), Collectors.toList())) {
-                                    tuples.add(new Tuple<String, Boolean>(matchingTenant, agr.resolvedActions(tenant.getAllowed_actions()).contains("kibana:saved_objects/*/write")));
+
+                                // find Wildcarded tenant patterns
+                                List<String> wildcardMatchingTenants = WildcardMatcher.from(tenant.getTenant_patterns()).getMatchAny(definedTenants.getCEntries().keySet(), Collectors.toList()) ;
```

Comment:
not a wildcard (*). 
But it is matching pattern configured in role, so better to rename `wildcardMatchingTenants` to original variable `matchingTenants` 

Also update comments accordingly.

Reply:
Hello @hardik-k-shah I have integrated the changes in https://github.com/chrousto/security/commit/a5683bf72d582c3930dce5e72c82769a70089482